lib/rules/no-return-await.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * @fileoverview Disallows unnecessary `return await`
+ * @author Jordan Harband
+ */
+"use strict";
+
+//------------------------------------------------------------------------------
+// Rule Definition
+//------------------------------------------------------------------------------
+
+module.exports = {
+    meta: {
+        docs: {
+            description: "disallow unnecessary `return await`",
+            category: "Best Practices",
+            recommended: false // TODO: set to true
+        },
+        fixable: false,
+        schema: [
+        ]
+    },
+
+    create(context) {
+        return {
+            ReturnStatement(node) {
+                if (node.argument && node.argument.type === "AwaitExpression") {
+                    context.report({
+                        node,
```

Comment:
Turns out if i pass `node.argument` i get a type of AwaitExpression, but if i getFirstToken i get Identifier - I think `node.argument` is the right value here? unless you mean to use `getFirstToken(node)` instead of `node.argument`?